test/e2e/ps_test.go

```diff
@@ -303,6 +303,22 @@ var _ = Describe("Podman ps", func() {
 		Expect(psAll.OutputToString()).To(Equal(psFilter.OutputToString()))
 	})
 
+	It("podman status filters should be inclusive", func() {
+		ctr := podmanTest.Podman([]string{"run", "-t", "-i", ALPINE, "ls", "/"})
+		ctr.WaitWithDefaultTimeout()
+		Expect(ctr.ExitCode()).To(Equal(0))
+
+		ctr2 := podmanTest.Podman([]string{"container", "create", ALPINE})
+		ctr2.WaitWithDefaultTimeout()
+		Expect(ctr2.ExitCode()).To(Equal(0))
+
```

Comment:
Can we add another "exited" container? I think we need one item that doesn't match the filter.